plan/physical.go

```diff
@@ -88,11 +86,9 @@ func validatePhysicalPlan(plan *Spec) error {
 		if validator, ok := pn.ProcedureSpec().(PostPhysicalValidator); ok {
 			return validator.PostPhysicalValidate(pn.ID())
 		}
-
 		if _, ok := pn.(*PhysicalPlanNode); !ok {
-			return errors.Errorf("logical node \"%v\" could not be converted to a physical node", pn.ID())
+			return fmt.Errorf("invalid physical query plan; found logical operation \"%v\"", pn.ID())
 		}
-
```

Comment:
I guess you could also add a check in here to check that the trigger is not `nil`.